src/pages/account-page/account-page.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react'
+import Box from '@mui/material/Box'
+import { useAuth } from '../../hooks/use-auth'
+import TabContext from '@mui/lab/TabContext'
+import TabPanel from '@mui/lab/TabPanel'
+import { SyntheticEvent, useState } from 'react'
+import { ContactInfo } from '../../components/Account-panel/Contact-info/Contact-info'
+import { BasicModal } from '../../components/Modal-custom-btn/ModalCustomBtnEdit'
+import {
+  StyledTab,
+  StyledTabs,
+  stBoxNav,
+  stContainerTabPanel,
+  stTabsNav,
+  stWavePink,
+  stBoxTabPanel
+} from './style'
+
+const AccountPage = () => {
+  const { isLoading, user } = useAuth()
+  const [value, setValue] = useState('1')
```

Comment:
this state should be `number` or `string`?

Reply:
Redone